tf2_py/CMakeLists.txt

```diff
@@ -1,157 +1,26 @@
 cmake_minimum_required(VERSION 3.5)
 project(tf2_py)
 
-## Find catkin macros and libraries
-## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
-## is used, also find other catkin packages
-find_package(catkin REQUIRED COMPONENTS rospy tf2)
-
-## System dependencies are found with CMake's conventions
-# find_package(Boost REQUIRED COMPONENTS system)
-
-
-find_package(PythonLibs 2 REQUIRED)
-include_directories(${PYTHON_INCLUDE_PATH} ${catkin_INCLUDE_DIRS})
-
-## Uncomment this if the package has a setup.py. This macro ensures
-## modules and global scripts declared therein get installed
-## See http://ros.org/doc/api/catkin/html/user_guide/setup_dot_py.html
-catkin_python_setup()
-
-#######################################
-## Declare ROS messages and services ##
-#######################################
-
-## Generate messages in the 'msg' folder
-# add_message_files(
-#   FILES
-#   Message1.msg
-#   Message2.msg
-# )
-
-## Generate services in the 'srv' folder
-# add_service_files(
-#   FILES
-#   Service1.srv
-#   Service2.srv
-# )
-
-## Generate added messages and services with any dependencies listed here
-# generate_messages(
-#   DEPENDENCIES
-#   std_msgs  # Or other packages containing msgs
-# )
-
-###################################
-## catkin specific configuration ##
-###################################
-## The catkin_package macro generates cmake config files for your package
-## Declare things to be passed to dependent projects
-## LIBRARIES: libraries you create in this project that dependent projects also need
-## CATKIN_DEPENDS: catkin_packages dependent projects also need
-## DEPENDS: system dependencies of this project that dependent projects also need
-catkin_package(
-#  INCLUDE_DIRS include
-#  LIBRARIES tf2_py
-  CATKIN_DEPENDS rospy tf2
-#  DEPENDS system_lib
-)
-
-###########
-## Build ##
-###########
-
-## Specify additional locations of header files
-## Your package locations should be listed before other locations
-# include_directories(include ${catkin_INCLUDE_DIRS} ${Boost_INCLUDE_DIRS})
-
-## Declare a cpp library
-# add_library(tf2_py
-#   src/${PROJECT_NAME}/tf2_py.cpp
-# )
-
-## Declare a cpp executable
-# add_executable(tf2_py_node src/tf2_py_node.cpp)
-
-## Add cmake target dependencies of the executable/library
-## as an example, message headers may need to be generated before nodes
-# add_dependencies(tf2_py_node tf2_py_generate_messages_cpp)
-
-## Specify libraries to link a library or executable target against
-# target_link_libraries(tf2_py_node
-#   ${catkin_LIBRARIES}
-# )
-
-
-# Check for SSE
-#!!! rosbuild_check_for_sse()
-
-# Dynamic linking with tf worked OK, except for exception propagation, which failed in the unit test.
-# so build with the objects directly instead.
+if(NOT WIN32)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14 -Wall -Wextra -Wpedantic")
```

Comment:
There's no way you could have known this, but we have changed this idiom.  See the top of i.e. https://github.com/ros2/turtlebot2_demo/blob/master/turtlebot2_drivers/CMakeLists.txt for the new idiom.